canonical/juju-dashboard
# WD-5941 - Display model name

## Done

- Replace model UUID with the model name in the tree view.

## QA

- Go to the advanced search page.
- Do a search.
- In the tree view the top level UUIDs should have been replaced with the model names.
- Clicking on a model name should take you to the model details page (which won't load yet due to the API issues).

## Details

https://warthogs.atlassian.net/browse/WD-5941

## Screenshots

<img width="419" alt="Screenshot 2023-08-24 at 12 05 53 pm" src="https://github.com/canonical/juju-dashboard/assets/361637/559b1d1f-0d9a-4558-b4e3-b05cffaf5f46">
